### PR TITLE
Update aws-sdk-go to v1.12.73

### LINKED ISSUE #59409: AWS SDK should be update to support `cn-northwest-1` region
Recently AWS released `cn-northwest-1` region in China, and the vendored AWS SDK by kubernetes is not updated to the latest official version, so the volume function can't used in `cn-northwest-1` region.

### PR DESCRIPTION
**Pull request by liubin**
Mainly to fix #59409 .

```release-note
Update aws-sdk-go to v1.12.73 #59635
```

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
 	"GoVersion": "go1.9",
-	"GodepVersion": "v79",
+	"GodepVersion": "v80",
```
Comment (lavalamp) [inline Godeps/Godeps.json]: Please don't change the godep version, or at least, do it in a separate PR and update all documentation, too.

File: vendor/github.com/aws/aws-sdk-go/aws/BUILD [hunk 2]
```diff
@@ -1,54 +0,0 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library")
```
Comment (lavalamp) [inline vendor/github.com/aws/aws-sdk-go/aws/BUILD]: Was the deletion of these BUILD files on purpose? (maybe run hack/update-bazel.sh?)

File: Godeps/Godeps.json (hunk 1 above)
Comment (thockin) [reply Godeps/Godeps.json]: agree

**Comment by thockin** [comment]
did you use out hack/* scripts to update deps?

**Comment by liubin** [comment]
@thockin @lavalamp May be I used an un-normal method to updated the vendor. Close it.
